Makefile

```diff
@@ -1,16 +1,16 @@
-all: flakes test
+all: pretty linted tests
 
-test:
-	python setup.py test
+pretty:
+	@python -m black .
```

Comment:
not a bad idea, but if you have [pre-commit](https://pre-commit.com/) correctly installed it will be passed locally. But it is a good idea to run it here as well.